returnn/datasets/postprocessing.py

```diff
@@ -0,0 +1,195 @@
+__all__ = ["PostprocessingDataset"]
+
+from dataclasses import dataclass
+from numpy import ndarray
+from typing import Any, Callable, Dict, Iterator, List, Optional, Tuple
+
+from returnn.tensor import TensorDict
+from returnn.util.basic import NumbersDict
+from .basic import Dataset, init_dataset
+from .util.strings import str_to_numpy_array
+
+
+@dataclass
+class LoadedData:
+    tensors: TensorDict
+    seq_idx: int
+
+
+Postprocessor = Callable[[TensorDict], TensorDict]
+StreamingPostprocessor = Callable[[Iterator[TensorDict]], Iterator[TensorDict]]
```

Comment:
It's only used once? Inline this below. (Even when used in multiple places, I think I would still inline this.)

Reply:
I found the type gets super long and not very readable. That's why I extracted them out.

Reply:
But it's still easier to read. Otherwise a user needs to lookup another place just to know the type.